pkg/remote/aws/lambda_function_details_fetcher.go

```diff
@@ -31,7 +31,7 @@ func (r *LambdaFunctionDetailsFetcher) ReadDetails(topic resource.Resource) (res
 	})
 	if err != nil {
 		logrus.Error(err)
-		return nil, remoteerror.NewResourceScanningError(err, resourceaws.AwsLambdaFunctionResourceType)
+		return nil, remoteerror.NewResourceScanningError(err, resourceaws.AwsLambdaFunctionResourceType, topic.TerraformId())
```

Comment:
Wow didn't see that in this details_fetcher we named the resource `topic` which is not right ... This was probably a copy/paste error from SNS ^^